lnd_multihopreceiverchainclaim_test.go

```diff
@@ -22,52 +24,63 @@ import (
 func testMultiHopReceiverChainClaim(net *lntest.NetworkHarness, t *harnessTest) {
 	ctxb := context.Background()
 
-	// First, we'll create a three hop network: Alice -> Bob -> Carol, with
-	// Carol refusing to actually settle or directly cancel any HTLC's
-	// self.
+	// First, we'll create a three hop network: Alice -> Bob -> Carol.
 	aliceChanPoint, bobChanPoint, carol :=
-		createThreeHopNetwork(t, net, true)
+		createThreeHopNetwork(t, net, false)
```

Comment:
Where is this ever set to `true` now?

Reply:
In all the unchanged tests still in `lnd_test.go` that use a hodl network.